Assets/MRTK/Tests/PlayModeTests/UserInputSimulationTest.cs

```diff
@@ -81,7 +80,48 @@ public IEnumerator HandsFreeInteractionTest()
         }
 
         [UnityTest]
-        public IEnumerator ArticulatedHandGestureTest()
+        public IEnumerator InputSimulationArtictulatedHandNearGrabbable()
+        {
+            var iss = PlayModeTestUtilities.GetInputSimulationService();
+            TestUtilities.PlayspaceToOriginLookingForward();
+            yield return null;
+
+            // No hands, default cursor should be visible
+            Assert.IsTrue(CoreServices.InputSystem.GazeProvider.GazeCursor.IsVisible, "Head gaze cursor should be visible");
+            //KeyInputSystem.StopKeyInputSimulation();
+            //yield return PlayModeTestUtilities.WaitForEnterKey();
+
+            // Begin right hand manipulation
+            KeyInputSystem.PressKey(iss.InputSimulationProfile.ToggleRightHandKey);
+            yield return null;
+            KeyInputSystem.AdvanceSimulation();
+            yield return null;
+
+            // Make sure right hand is tracked
+            Assert.True(iss.HandDataRight.IsTracked);
+
+            // Head cursor invisible when hand is tracked
+            Assert.IsFalse(CoreServices.InputSystem.GazeProvider.GazeCursor.IsVisible, "Eye gaze cursor should not be visible");
+
+            // Create grabbable cube
+            var cube = GameObject.CreatePrimitive(PrimitiveType.Cube);
+            cube.AddComponent<NearInteractionGrabbable>();
+            cube.transform.position = new Vector3(0, 0, 1.2f);
+            yield return null;
+
+            // Grab pointer is near grabbable
+            TestHand hand = new TestHand(Handedness.Right);
+            var grabPointer = hand.GetPointer<SpherePointer>();
+            Assert.IsTrue(grabPointer.IsNearObject, "Grab pointer should be near a grabbable");
+
+            yield return PlayModeTestUtilities.WaitForInputSystemUpdate();
+
+            // Head cursor invisible when grab pointer is near grabbable
```

Comment:
Would be good to make sure that hand grabbabel pointer is enabled here (so you can grab stuff). Also would be good to check that hand rays are visible when hand is not near grabbable. So we should have:

1. hand rays are enabled when not near grabbable
2. gaze cursor not visible.
3. When near grabbable, rays are off and the gaze cursor not visible.